modules/monitoring-setting-up-pod-topology-spread-constraints-for-prometheus.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-up-pod-topology-spread-constraints-for-prometheus_{context}"]
+= Setting up pod topology spread constraints for Prometheus
+
+For core {product-title} platform monitoring, you can set up pod topology spread constraints for Prometheus to fine tune how pod replicas are scheduled to nodes across zones.
+Doing so helps ensure that Prometheus pods are highly available and run more efficiently because workloads are spread across nodes in different data centers or hierarchical infrastructure zones.
+
+You configure pod topology spread constraints for core platform monitoring in the `cluster-monitoring-config` config map.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add  values for the following settings under `data/config.yaml/prometheusK8s` to configure topology spread constraints:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      topologySpreadConstraints:
+      - maxSkew: 1 <1>
+        topologyKey: monitoring <2>
+        whenUnsatisfiable: DoNotSchedule <3>
+        labelSelector: 
+          matchLabels: <4>
+            app.kubernetes.io/name: prometheus
+----
+<1> Specify a numeric value for `maxSkew`, which defines the degree to which pods can be unevenly distributed. 
```

Comment:
`can be` changed to `may be` is better
https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#spread-constraint-definition

Reply:
Changed to "might be". The IBM Style Guide does not allow us to use "may" when we mean "might."